polarCoordinateTraversalTest.m

```diff
@@ -278,15 +278,211 @@ function testRayGoesAlongThetaBoundary(testCase)
         verifyEqual(testCase, thetaVoxels, expected_thetaVoxels);
 end
 
-% TODOs
 % - Ray intersects a “theta boundary” outside the circle, and ensure it does 
 %   NOT record this as a theta hit.
+function testRayInterSectThetaBoundaryOutsideTheCircle(testCase)
```

Comment:
```testRayIntersect...```
```Intersect``` is one word, so no need to capitalize to ```InterSect```.

Reply:
Ooof aight let me fix that